docs/install.rst

```diff
@@ -3,121 +3,120 @@
 Installation
 ============
 
-Here is a step by step plan on how to install Read the Docs.
+Here is a step by step guide on how to install Read the Docs.
 It will get you to a point of having a local running instance.
 
 .. warning::
 
-    Read the Docs does not itself run under Python 3 (though it does support
-    building documentation for Python 3 projects). Please ensure the subsequent
-    steps are performed using Python 2.7.
+    Read the Docs does not full support running under Python 3
```

Comment:
The rtd code can run on python3, but I think we are not confident enough (#3570)

Reply:
I wouldn't advertise any level of Python 3 support until we use it on production, have a running test suite, and bandwidth to response user reports. I think it would be better to keep the old wording or say something like "Python 3 support is planned, PRs welcome."

Reply:
Yup, I don't think there is any reason to scare users into not using py3. We'll be using it in prod as soon as we migrate. I'm not sure I even want to mention that RTD isn't py3 in prod, as I'd rather we get PRs now for py3 incompatibilities.